### DISCUSSION TITLE
DigitalOcean droplet configure domain problem

### DISCUSSION THREAD

**Discussion by AlexNasonov** [requarks/wiki#3397] [Help / Questions]
**Question**
Created a droplet from DO Marketplace and assigned HTTP domain. It works fine if I access directly via IP and crushes on access via domain (or subdomain). I've got two A entries in DNS like: my-domain.com and wiki.my-domain.com. Both pointing on the droplet. Made relevant setting in Wiki.JS Administration setup. Can't make it work as it's aught to. Request doesn't recieve response.

**Host Info:**
OS: [Ubuntu 18.04, Docker]
Wiki.js version: [2.3.81]

**Comment by ghost** [comment]
From your question, I would like to know more about: 
1. are you using docker?
2. what web server are u using?
3. if u create a simple index.html page, are u able to reach that said index.html page via wiki.my-domain.com
4. if number 1 to 3 works, then we proceed to check if you have done proxy passing to the said docker container.
